### PR TITLE
docs: update GOVERNANCE

### PR DESCRIPTION
**Pull request by tiagolobocastro**

### CODE REVIEW

**Review by orville-wright — CHANGES_REQUESTED** [review]
@tiagolobocastro - Great that you are making suggested changes to the GOVERNANCE doc, as discussed in the meeting with Jorge Castro .

Some thoughts here...
- The Governance doc is the most powerful & important doc of our org.
- it needs special care & treatment every time we make a change to it.

Its great that we're following good governance process here, IMHO if / when we change our governance doc, we need a special process so that all maintainers can understand the suggested changes (in a closed safe session), understand the rationale behind it and the benefits to the community, project and CNCF.

With this in mind, I recommend the steps to change our Governance doc are...

1. You create an GitHub ISSUE in the community Repo, recommending the Maintainers consider your proposal to change the Governance doc. (you don't need to detail your changes in the doc). We need an ISSUE as it is our core reference to this possible event.
3. We have a closed Maintainers discussion session. You propose your suggested changes. You help us all get aligned on them, as well as identify any miss-understandings and refine the language.
4. We a vote on your GitHub ISSUE.
5. If the ISSUE is approved... you create a PR against the governance file and link it to the GitHub ISSUE.; and you make the approved changes in the PR.
6. The Maintainers & community can comment & vote in public... to approve/reject the PR accordingly.  This is a special PR and outside the regular PR approval process.
7. Only Maintainers can approve and Merge a PR. Community feedback is openly considered by maintainers. The Maintainers may need to meet in order to achieve consensus on approving a governance doc change PR.
8. If the PR is approved, you Merge the PR & the changes are executed. & we now have a full audit chain to track the change to our governance doc.

**Note**: 
- Binding votes (Maintainers only) are the only votes that are counted for ISSUE votes and changes to Governance docs. Community votes are not binding votes.
- This is why we must have an ISSUE vote first. A PR has a different more relaxed approval process where only a 'reasonable number of reviewers' need to approve a PR. Which is too unsafe for a change to our Governance doc
- IMHO, we should also require a **SuperMajority** in order to make changes to the Governance doc. / This would require a 67% approval vote threshold to be achieved.

As for your PR...
I think jumping to the PR step first, is a accelerating the process a little bit too quickly. As is taking general public feedback/vote in the PR "Review comments" where the Maintainers haven't had a chance to review the conceptual changes first... and refine the language  (in a closed safe session).

- For Governance doc changes, we need to be extremely carful & vigilant.

I recommend we put this PR on hold until we achieve steps 1, 2, 3 & 4.

**Comment by tiagolobocastro** [comment]
> 1. You create an GitHub ISSUE in the community Repo, recommending the Maintainers consider your proposal to change the Governance doc. (you don't need to detail your changes in the doc). We need an ISSUE as it is our core reference to this possible event.

Another suggestion is to do it on a PR so everyone can comment inline, rather than a plain issue. I'm not sure how we can convey changes effectively on an issue, perhaps a PR would be better suited.

> * This is why we must have an ISSUE vote first. A PR has a different more relaxed approval process where only a 'reasonable number of reviewers' need to approve a PR. Which is too unsafe for a change to our Governance doc

As long as the same rule is in place, not sure what makes it unsafer. I'd argue a PR is safer, because reviewers would get to see whatever is being modified as opposed to reaching some kind of consensus only for it to diverge in the actual PR.
Another approach could be an RFC-line process.

> As for your PR... I think jumping to the PR step first, is an accelerating the process a little bit too quickly. As is taking general public feedback/vote in the PR "Review comments" where the Maintainers haven't had a chance to review the conceptual changes first... and refine the language (in a closed safe session).

Ok... 

> 
> * For Governance doc changes, we need to be extremely carful & vigilant.
> 
> I recommend we put this PR on hold/draft until we achieve steps 1, 2, 3 & 4.

Which is why I raised it as a draft to begin with 👍

**Comment by tiagolobocastro** [comment]
Closing until we have an established procedure to update the Governance

**Pull request closed without merging**
